doc/user_guide/interactions/parameters.rst

```diff
@@ -0,0 +1,485 @@
+An *interval* selection allows you to select chart elements by clicking and dragging.
+You can create such a selection using the :func:`selection_interval` function:
+
+.. altair-plot::
+
+   interval = alt.selection_interval()
+   make_example(interval)
+
+As you click and drag on the plot, you'll find that your mouse creates a box
+that can be subsequently moved to change the selection.
+
+The :func:`selection_interval` function takes a few additional arguments; for
+example we can bind the interval to only the x-axis, and set it such that the
+empty selection contains none of the points:
+
+.. altair-plot::
+
+   interval_x = alt.selection_interval(encodings=['x'], empty=False)
+   make_example(interval_x)
+
+Point Selections
+^^^^^^^^^^^^^^^^
+A *point* selection allows you to select chart elements one at a time
+via mouse actions. By default, points are selected on click:
+
+.. altair-plot::
+
+    point = alt.selection_point()
+    make_example(point)
+
+By changing some arguments, we can select points when hovering over them rather than on
+click. We can also set the ``nearest`` flag to ``True`` so that the nearest
+point is highlighted:
+
+.. altair-plot::
+
+    point_nearest = alt.selection_point(on='pointerover', nearest=True)
+    make_example(point_nearest)
+
+Point selections also allow for multiple chart objects to be selected.
+By default, chart elements can be added to and removed from the selection
+by clicking on them while holding the *shift* key, you can try in the two charts above.
+
+Selection Targets
+~~~~~~~~~~~~~~~~~
+
+For any but the simplest selections, the user needs to think about exactly
+what is targeted by the selection, and this can be controlled with either the
+``fields`` or ``encodings`` arguments. These control what data properties are
+used to determine which points are part of the selection.
+
+For example, here we create a small chart that acts as an interactive legend,
+by targeting the Origin field using ``fields=['Origin']``. Clicking on points
+in the upper-right plot (the legend) will propagate a selection for all points
+with a matching ``Origin``.
+
+.. altair-plot::
+
+    selection = alt.selection_point(fields=['Origin'])
+    color = alt.condition(
+        selection,
+        alt.Color('Origin:N').legend(None),
+        alt.value('lightgray')
+    )
+
+    scatter = alt.Chart(cars).mark_point().encode(
+        x='Horsepower:Q',
+        y='Miles_per_Gallon:Q',
+        color=color,
+        tooltip='Name:N'
+    )
+
+    legend = alt.Chart(cars).mark_point().encode(
+        alt.Y('Origin:N').axis(orient='right'),
+        color=color
+    ).add_params(
+        selection
+    )
+
+    scatter | legend
+
+The above could be equivalently replace ``fields=['Origin']`` with
+``encodings=['color']``, because in this case the chart maps ``color`` to
+``'Origin'``. Also note that there is a shortcut to create interactive legends in Altair
+described in the section :ref:`legend-binding`.
```

Comment:
```suggestion
Alternatively, we could express ``fields=['Origin']`` as ``encodings=['color']``, because our chart maps ``color`` to
``'Origin'``. Also note that there is a shortcut to create interactive legends in Altair
described in the section :ref:`legend-binding`.
```